ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -221,8 +221,10 @@ spec:
       params:
         - name: pipeline_image
           value: "$(params.pipeline_image)"
-        - name: operator_path
-          value: "$(tasks.bundle-path-validation.results.package_path)"
+        - name: affected_operators
+          value: "$(tasks.bundle-path-validation.results.package_name)"
+        - name: affected_catalog_operators
+          value: "$(tasks.detect-changes.results.affected_catalog_operators)"
```

Comment:
Again, the `detect-changes` task is not a part of this pipeline. Maybe we should include it to make this easier?

Reply:
This duplicates Aleš's comment, disregard.